application/client/src/app/ui/elements/transport/setup/udp/multicast/template.html

```diff
@@ -1,19 +1,21 @@
-<mat-form-field class="inline">
-    <input matInput placeholder="Multicast Group Address" [(ngModel)]="multicast.multiaddr" (ngModelChange)="_ng_onChanges()" [errorStateMatcher]="errors.multiaddr">
-    <mat-error *ngIf="errors.multiaddr.getErrorCode() === 'INVALID'">
-        Please enter a valid IPv4/IPv6 address
-    </mat-error>
-    <mat-error *ngIf="errors.multiaddr.getErrorCode() === 'REQUIRED'">
-        <strong>required</strong>
-    </mat-error>
-</mat-form-field>
-<mat-form-field class="inline">
-    <input matInput placeholder="Network Interface" [(ngModel)]="multicast.interface" (ngModelChange)="_ng_onChanges()" [errorStateMatcher]="errors.interface">
-    <mat-error *ngIf="errors.interface.getErrorCode() === 'INVALID'">
-        Please define valid interface
-    </mat-error>
-    <mat-error *ngIf="errors.interface.getErrorCode() === 'REQUIRED'">
-        <strong>required</strong>
-    </mat-error>
-</mat-form-field>
-
+<div class="form-fields">
```

Comment:
could we refuse of `<div class="form-fields">` as container? and use root-container `app-transport-udp-multicast`